wikicharts/wikicharts.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #402.            last_x = self.df[str(x)].iat[-1]
</pre><p>Out of curiosity, why is it necessary to use <code>str(x)</code>? x should already be an appropriate data type for indexing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>